Engine/Filesystem/Path.cpp

```diff
@@ -250,9 +250,20 @@ std::string Path::GetFullPath() const
 	return file_path;
 }
 
+std::string Path::GetFilename(const std::string& path)
+{
+	return path.substr(path.find_last_of('/') + 1, -1);
+}
+
+std::string Path::GetFilenameWindows(const std::string& path)
+{
+	return path.substr(path.find_last_of('\\') + 1, -1);
+}
```

Comment:
Functions should be const right?

Reply:
Because they are static they cannot be const

Reply:
my bad didnt realize